bindings/node/lib/autoEncrypter.js

```diff
@@ -90,6 +93,23 @@ module.exports = function (modules) {
      *
      * await client.connect();
      * // From here on, the client will be encrypting / decrypting automatically
+     * @example <caption>Create an AutoEncrypter that makes use of libmongocrypt's CSFLE shared library</caption>
+     * // Enabling autoEncryption via a MongoClient using CSFLE shared library
+     * const { MongoClient } = require('mongodb');
+     * const client = new MongoClient(URL, {
+     *   autoEncryption: {
+     *     kmsProviders: {
+     *       aws: {...}
```

Comment:
```suggestion
     *       aws: {}
```

The empty object was intentional b/c its how auto credential fetching looks